src/testing/puppeteer/puppeteer.d.ts

```diff
@@ -0,0 +1,3 @@
+/**
```

Comment:
Non-blocking question I missed the first time around - do we need this file?

Reply:
I have no idea how this got in here 😆 removing it.